### ISSUE TITLE
RayTracing shaders compilation is not yet support

### ISSUE THREAD

**Issue by unknownue** [Ralith/vk-shader-macros#6]
Currently, the following code would not compile:
```rust
let spv = include_glsl!("triangle.rgen.glsl", kind: rgen);
// or
let spv = include_glsl!("triangle.rgen.glsl");
// or
let spv = include_glsl!("triangle.rgen");
```

Test shaders can be found at [here](https://github.com/nvpro-samples/vk_raytracing_tutorial_KHR/tree/master/ray_tracing__simple/shaders).

**Comment by Ralith** [comment]
Thanks for bringing this to my attention! Does #7 solve the issue for you?

**Comment by unknownue** [comment]
I have tried #7, the compilation does work, but the corresponding shader does not compile successfully. Report that `#extension GL_EXT_ray_tracing is not supported.`

I noticed that current [shaderc-rs](https://github.com/google/shaderc-rs/tree/master/shaderc-sys/build) is using the [libshaderc](https://github.com/google/shaderc/) updated at 2020.3. It seems that the `VK_KHR_ray_tracing` had not been released at 2020.3. If I change `GL_EXT_ray_tracing` to `GL_NV_ray_tracing` as well as other `NV` naming, the shader does compile.

Because `shaderc-rs` would link local `libshaderc` package by default(if not found, it will build from source), I also tried to install latest `libshaderc` package, remove `shaderc-rs` build cache, and rebuild everything. The shader finally compile successfully using `GL_EXT_ray_tracing`.

**Comment by unknownue** [comment]
The issue can be closed now, or wait until `shaderc-rs` update its dependency.

Note: I have not tested mesh/asm shader.

**Comment by Ralith** [comment]
Thanks for testing!

**Comment by Ralith** [comment]
Released in v0.2.6.
